### PR TITLE
NTR - add elasticsearch version

### PR DESCRIPTION
**Pull request by PascalThesing**
Add the documentation for the new elasticsearch version config.

### CODE REVIEW

File: source/sysadmins-guide/elasticsearch-setup/index.md [hunk 1]
```diff
@@ -87,7 +89,7 @@ return [
 ];
 ```
 
-Shopware 5 communicates with Elasticsearch using the latter's REST API. The `hosts` array accepts multiple address syntaxes, about which you can read more [here](https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_configuration.html#_host_configuration). The `number_of_shards` and `number_of_replicas` parameter provided to the generated index. A `null` configuration allows to use the elastic search server configuration of this parameters.
+Shopware 5 communicates with Elasticsearch using the latter's REST API. The `hosts` array accepts multiple address syntaxes, about which you can read more [here](https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_configuration.html#_host_configuration). The version defines the used Elasticsearch Version. If not set in the config, Shopware will detect it automatically. The `number_of_shards` and `number_of_replicas` parameter provided to the generated index. A `null` configuration allows to use the elastic search server configuration of this parameters.
```
Comment (mitelg) [inline source/sysadmins-guide/elasticsearch-setup/index.md]: maybe you should state here, why should one set the version in the config -> performance, etc

**Comment by mitelg** [comment]
so this is ok @soebbing ?

**Comment by PascalThesing** [comment]
I think so @mitelg.

**Comment by mitelg** [comment]
thanks @PascalThesing :blush:

**Pull request merged**